modules/Movecraft/src/main/resources/types/Airskiff.craft

```diff
@@ -0,0 +1,120 @@
+name: Airskiff
+maxSize: 80
+minSize: 10
+allowedBlocks:
+    - "#planks"
+    - water
+    - lava
+    - "#logs"
+    - "#glass"
+    - dispenser
+    - note_block
+    - "#beds"
+    - sticky_piston
+    - piston
+    - piston_head
+    - sticky_piston
+    - piston
+    - piston_head
+    - "#wool"
+    - gold_block
+    - "#slabs"
+    - bookshelf
+    - torch
+    - fire
+    - "#stairs"
+    - "#chests"
+    - redstone_wire
+    - diamond_block
+    - crafting_table
+    - furnace
+    - blast_furnace
+    - smoker
+    - "#signs"
+    - "#doors"
+    - ladder
+    - lever
+    - "#pressure_plates"
+    - redstone_torch
+    - redstone_wall_torch
+    - "#buttons"
+    - "#fences"
+    - cake
+    - repeater
+    - "#trapdoors"
+    - iron_bars
+    - "#fence_gates"
+    - brewing_stand
+    - cauldron
+    - redstone_lamp
+    - ender_chest
+    - tripwire_hook
+    - tripwire
+    - emerald_block
+    - beacon
+    - flower_pot
+    - "#heads"
+    - "#anvil"
+    - comparator
+    - daylight_detector
+    - redstone_block
+    - hopper
+    - "#rails"
```

Comment:
unless I'm very much mistaken this is a behaviour change

Reply:
My bad, this was unintended